packages/mobile/src/navigator/NavigatorWrapper.tsx

```diff
@@ -225,6 +261,33 @@ const styles = StyleSheet.create({
     right: 0,
     bottom: 0,
   },
+  modal: {
+    margin: 0,
+  },
+  shakeForSupport: {
+    position: 'absolute',
+    bottom: 0,
+    height: 240,
```

Comment:
We should avoid hardcoding the `height`, and let the content define it, so there's less risk that the layout breaks on devices with a narrow screen, or if the font size changes.